os/hardening-guide.md

```diff
@@ -58,4 +58,4 @@ systemctl daemon-reload
 ```
 
 [sshd-guide]: customizing-sshd.md
-[etcd-sec-guide]: https://github.com/coreos/etcd/blob/v3.1.5/Documentation/op-guide/security.md
+[etcd-sec-guide]: https://github.com/coreos/etcd/blob/v3.2.11/Documentation/op-guide/security.md
```

Comment:
Is this something we'll have to remember to update along with etcd?  Can it link to https://coreos.com/etcd/docs/latest/op-guide/security.html ?

Reply:
I left it as a link to the tag pursuant to the discussion in https://github.com/coreos/docs/pull/1040.